pyproject.toml

```diff
@@ -33,9 +33,9 @@ lupa = "*"
 fakeredis = "*"
 
 [tool.flake8]
-# B006: Do not use mutable data structures for argument defaults. They are created during function definition time. All calls to the function reuse this one instance of that data structure, persisting changes between them.
```

Comment:
I think it is better not to ignore rules for no particular reason.

Reply:
Commented rules are mistakenly ignored. Thanks for guiding me.